USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs

```diff
@@ -675,6 +689,13 @@ private string RenderMarker(Marker input)
                 case IDMarker idMarker:
                     currentChapterLabel = null;
                     break;
+                case TOC3Marker tocMarker:
+                    string name = tocMarker.BookAbbreviation;
+                    output.Append("<div class=\"header\">"); // same as header format - nicer when navigate
+                    output.AppendLine($"<div class=\"toc3-ref\" id=\"{name}\">{name}</div>");

```

Comment:
The style.css should update as below, so that the toc reference can be in the same line with header text

```
.header {
    ...
    display: inline-block;
}

.toc3-ref {
    display: inline;
    visibility: hidden;
}
```